src/custom/components/Footer/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import Version from '../Version'
+const FOOTER_URLS = [
+  { name: '© 2021 Gnosis' },
```

Comment:
do we want this?

Reply:
And is it really copyright? (that's what the © means, right?)
I thought it was an opensource fork of an opensource project.

Reply:
I asked anna, if we want "🐮 Powered by Gnosis Protocol"